tests/unit/test_utils/test_vdb/test_elastic_vdb.py

```
@@ -370,17 +354,15 @@ def test_check_collection_exists(
             index="test_collection"
         )
 
-    @patch("nvidia_rag.utils.common.get_config")
     @patch("nvidia_rag.utils.vdb.elasticsearch.elastic_vdb.Elasticsearch")
     @patch("nvidia_rag.utils.vdb.elasticsearch.elastic_vdb.VectorStore")
     def test_get_collection(
-        self, mock_vector_store, mock_elasticsearch, mock_get_config
-    ):
+        self, mock_vector_store, mock_elasticsearch):
```

Ruff (0.14.5)

Unused method argument: `mock_vector_store`

(ARG002)